yt_dlp/extractor/jwplatform.py

```diff
@@ -35,6 +41,16 @@ def _extract_embed_urls(cls, url, webpage):
         mobj = re.search(r'<div\b[^>]* data-video-jw-id="([a-zA-Z0-9]{8})"', webpage)
         if mobj:
             return [f'jwplatform:{mobj.group(1)}']
+        # iltalehti.fi
+        mobj = re.search(r'<script>\s*window.App\s*=\s*({.*})\s*</script>', webpage)
```

Comment:
This should only be checked in that specific site, correct? In that case, create a new extractor

Reply:
Ok, will do.